src/screens/login/components/login-form/login-form.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { TextInput } from 'react-native-paper';
+import { View } from 'react-native';
+import { Formik } from 'formik';
+
+import { styles } from '../../styles';
+import { MainButton } from 'src/components';
+import { AppColor } from 'src/common/enums';
+import { defaultLoginPayload } from '../../common';
+import { authenticateUserValidationSchema } from 'src/validation-schemas';
+import { FormInput } from 'src/components';
+import { IAuthenticateUser } from 'src/common/types';
+
+const LoginForm: React.FC = () => {
+  const [secure, setSecure] = useState(true);
+
+  const onLoginPressed = (values: IAuthenticateUser) => {
+    // TODO authentication logic
+    console.log('Received data from the form: ', values);
+  };
```

Comment:
Add further layers: redux and service. This TODO must be placed instead of http request.